install/helm/open-match/templates/om-configmap-override.yaml

```diff
@@ -31,6 +31,6 @@ data:
         grpcport: "{{ .Values.evaluator.grpcPort }}"
         httpport: "{{ .Values.evaluator.httpPort }}"
     synchronizer:
-      registrationIntervalMs: 3000ms
-      proposalCollectionIntervalMs: 15000ms
+      registrationIntervalMs: 250ms
```

Comment:
can you make the following change to https://github.com/googleforgames/open-match/blob/master/Makefile#L376 or give the CI environment a larger registration interval to make our CI happy? The ignoreListTTL should be in sync with the one in https://github.com/googleforgames/open-match/blob/master/internal/statestore/testing/consts.go#L29. Our previous registrationInterval didn't catch this discrepancy but the new one does.

```diff
- --set open-match-core.ignoreListTTL=1000ms \
+ --set open-match-core.ignoreListTTL=500ms \
```